.lando.yml

```diff
@@ -0,0 +1,15 @@
+name: starshot
+recipe: drupal10
+config:
+  webroot: web
+tooling:
+  drupal:install:
+    service: appserver
+    cmd: |
+      composer install
+      composer drupal:install
+      echo ""
+      echo "🚀 Board the spaceship!"
+      echo "👇 Click on the link below to login 👇"
+      drush uli -l 'https://starshot.lndo.site'
```

Comment:
So, rather than have to pass `-l` or `--uri`, could we do a similar thing to what we're doing with DDEV and inject a `DRUSH_OPTIONS_URI` environment variable into the app server that has the same value as `https://$LANDO_DOMAIN`? That way, Drush will _always_ have the correct URI when run within the app server container and we wouldn't have to remember to pass it.

Reply:
Yes we can pass the `DRUSH_OPTIONS_URI` in Lando like this: 
```yaml
services:
  appserver:
    overrides:
      environment:
        DRUSH_OPTIONS_URI: "http://mysite.lndo.site"
```

But it does not work because we have a `drush.yml` file committed with a `localhost` value - and it takes precedence over the environment variables (see [previous comment](https://github.com/phenaproxima/starshot-prototype/pull/55#discussion_r1597130590)).

Reply:
@phenaproxima do we _need_ the drush.yml file committed?

If no, then we can use the env var for Drush URI.

If yes, should I add [the `echo` to rewrite it ](https://github.com/phenaproxima/starshot-prototype/pull/55/files/ec0c4ac9c83dcba4d798be2991d1ae0eb9b97a32#diff-307b4548b91e997f23c35d90ed5e9a044e156b90425c9923d6b85d7dc11e67e2R20) again?